ocular_cli/src/commands/chains/registry_list.rs

```diff
@@ -1 +1,34 @@
+use crate::prelude::*;
+use abscissa_core::{Command, Runnable};
+use clap::Parser;
+use serde::Serialize;
 
+#[derive(Command, Debug, Parser)]
+pub struct RegistryListCmd {}
+
+impl Runnable for RegistryListCmd {
+    /// List all chains in local config file
+    fn run(&self) {
+        abscissa_tokio::run(&APP, async {
+            match ocular::chain::registry::list_chains().await {
+                Ok(mut info) => {
+                    info.drain(0..1);
+                    for info in info {
+                        let info = info.as_str();
+                        let buf = Vec::new();
+                        let formatter = serde_json::ser::PrettyFormatter::with_indent(b"    ");
+                        let mut serialize = serde_json::Serializer::with_formatter(buf, formatter);
+                        info.serialize(&mut serialize).unwrap();
```

Comment:
I'm not sure what this is accomplishing; I don't see any formatting changes when I run the command. I think it can be simplified to an iterator filter and foreach for printing.

Reply:
Did you remove it before coming to this conclusion?

Reply:
Ooh, I didn't see this part:

```
I think it can be simplified to an iterator filter and foreach for printing
```
I thought you meant we should continue with `serde_json::to_string_pretty` because that doesn't print very good. However, since we already use the above for the `show` command, I'll make it a function to avoid repetition